handlers/templates.go

```diff
@@ -0,0 +1,52 @@
+package handlers
+
+import (
+	"html/template"
+	"io/ioutil"
+	"path"
+	"strings"
+)
+
+// TemplatesFromAssets returns the HTML templates from the assets filesystem.
+// It has a signature that allows it to be used with templates.Must.
+func TemplatesFromAssets() (*template.Template, error) {
+	t := template.New("_base_")
+	d, err := Assets.Open("/")
+	if err != nil {
+		return t, err
+	}
+	fis, err := d.Readdir(0)
```

Comment:
@fawick I'd prefer the slightly longer but much more human readable `files` in place of `fis`

Reply:
Done